README.md

```diff
@@ -1,2 +1,249 @@
+
+```
+$ conda-env-tracker ${ENV_REPO}
+Pushing changes to manifest/default
+Pushing changes to manifest/legacy
+```
+
+Tag the newly resolved environments (phase 2):
+
+```
+$ conda-env-tracker-timestamp ${ENV_REPO}
+Pushing tag env-default-2015_11_12-1
+Pushing tag env-legacy-2015_11_12
+```
+
+And deploy (phase 3):
+
+```
+$ conda-env-tracker-deploy ${ENV_REPO} /path/to/install/environments
+Fetching package metadata: .
+Fetching libgfortran-1.0-0
+Fetching numpy-1.10.1-py35_0
+Fetching openblas-0.2.14-3
+Linking default/latest to env-default-2015_11_12-1
+Fetching numpy-1.10.1-py27_0
+Fetching pip-7.1.2-py27_0
+Fetching python-2.7.10-2
+Fetching setuptools-18.4-py27_0
+Fetching wheel-0.26.0-py27_1
+Linking legacy/latest to env-legacy-2015_11_12
+```
+
+Finally, we can see that our environments have been deployed as expected:
+
+```
+$ ls -ltr /path/to/install/environments/*
+/path/to/install/environments/defaults:
+drwxr-xr-x pelson 4096 Nov 12 10:54 2015_11_12
+drwxr-xr-x pelson 4096 Nov 12 11:31 2015_11_12-1
+lrwxrwxrwx pelson   12 Nov 12 11:31 latest -> 2015_11_12-1
+
+/path/to/install/environments/legacy:
+drwxr-xr-x pelson 4096 Nov 12 11:32 2015_11_12
+lrwxrwxrwx pelson   10 Nov 12 11:32 latest -> 2015_11_12
+```
+
+And can verify that we have the Python we expected:
+
+```
+$ /path/to/install/environments/latest/bin/python --version
+Python 2.7.10
+```
+
+Phase 4: More labels (labeltag)
+===============================
+
+As environments evolve it is desirable to be able to label those of value.
+
+As we have already seen, the "latest" label is automatically added to point to the
+tag which points to the most recent commit. Labelled environments are implemented as
+symlinks which point to the appropriate tag at deploy time.
+
+It is possible to add other labels which point to tagged environments. Labels are currently
+defined in the environment definition branch, under a directory called labels. For instance, to
+add a "pre-prod" label to our default environment definition:
+
+```
+git checkout default
+mkdir labels
+echo "env-default-2015_11_12-1" > labels/pre-prod.txt
+git add labels
+git commit -am "Added a pre-prod label."
+```
+
+Re-running the deployment will see a new symbolic linked environment, pointing to the appropriate tag.
+
+There is some machinery which helps us move through a next -> current -> previous workflow, but this is
+likely to change in the future. Please raise an issue if you would like more detail on this.
+
+Notes
+-----
+
+* If using ``conda-env-tracker`` on a local git repository, it will not be possible to push changes to a branch which is checked out.
+  In these situations it is safest to put your local repo into "detached head" mode (one option ``git checkout --detach``).
+
+* Assumes a basic approach of merge and fix, rather than verify before merge. To mitigate this concern, the labels concept allows
+  us to point to a tagand would not break if we merged something erroneously.
```

Comment:
> In truth that whole sentence doesn't make a whole lot of sense

Maybe it's not the easiest to read, but it does make sense ... at least to me. :wink: